### PR TITLE
Add Elastic Stack data link integration

### PR DESCRIPTION
**Pull request by bjsignalfx**
Added Elastic Stack to Linked Services.

Updated installation text for all Linked Services.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,10 +3,12 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: golang:1.12
+      - image: golang:1.13
     steps:
       - checkout
-      - run: go get -u github.com/raviqqe/liche
+        # Do hacky things to install a tool with go modules.
```
Comment (alstory) [inline .circleci/config.yml]: is: Do hacky things to install a tool with go modules.  
change to: Use unconventional methods to install a tool with Go modules.

File: collectd-uptime/README.md [hunk 2]
```diff
@@ -43,10 +43,6 @@ There are no configuration options for this plugin.
 
 This plugin provide the time that an individual host has been up since last boot.
```
Comment (alstory) [inline collectd-uptime/README.md]: is: This plugin provide
change to: This plugin provides

File: .circleci/config.yml (hunk 1 above)
Comment (keitwb) [reply .circleci/config.yml]: This is for developers only -- it is not user-facing.

Comment (keitwb) [reply .circleci/config.yml]: Also @bjsignalfx this shouldn't even be in this PR.  Make sure you have rebased from the latest master.

Comment (bjsignalfx) [reply .circleci/config.yml]: @keitwb I branched off release then cherry picked my changeset from master.

**Pull request closed without merging**